backend/typescript/services/interfaces/IReviewService-ti.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * This module was automatically generated by `ts-interface-builder`
+ */
+import * as t from "ts-interface-checker";
+// tslint:disable:object-literal-key-quotes
+
+export const Author = t.iface([], {
+  fullName: "string",
+  displayName: t.opt("string"),
+  attribution: t.opt("string"),
+});
+
+export const Publisher = t.iface([], {
+  fullName: "string",
+  publishYear: "number",
+});
+
+export const Format = t.iface([], {
+  format: "string",
+  price: "string",
+  isbn: "string",
+});
+
+export const Book = t.iface([], {
+  title: "string",
+  titlePrefix: t.opt("string"),
+  seriesOrder: t.opt("string"),
+  illustrator: t.opt(t.array("string")),
+  translator: t.opt(t.array("string")),
+  formats: t.array("Format"),
+  minAge: "number",
+  maxAge: "number",
+  authors: t.array("Author"),
+  publishers: t.array("Publisher"),
+  seriesName: t.opt("string"),
+});
+
```

Comment:
Can this file be moved to a different folder titled interfaceCheckers or something?